query/functions/structure.feature

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#noinspection CucumberUndefinedStep
+Feature: Tests for various shapes of function bodies
```

Comment:
Don't need to write `Tests for...`! We don't have that anywhere else? Cue - filename. I'd call it `Feature: Function Body Structure`